.github/workflows/testingfarm.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: Testing farm tests
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  testingfarm:
+    name: "Run in testing farm"
+    runs-on: ubuntu-latest
+    if: |
+      github.event.issue.pull_request
+      && contains(github.event.comment.body, '[test]')
+      && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4
+      with:
+        ref: "refs/pull/${{ github.event.issue.number }}/head"
+        submodules: true
+
+    - name: Run the tests
+      uses: sclorg/testing-farm-as-github-action@main
+      with:
+        compose: Fedora-39
+        api_key: ${{ secrets.TF_API_KEY }}
+        git_url: "https://gitlab.cee.redhat.com/rhinstaller/builders"
+        git_ref: "master"
+        path: "ansible"
+        tmt_plan_regex: "github-kstest-setup"
+        pull_request_status_name: "Testing farm"
+        tf_scope: private
```

Comment:
Based on the information from Miro, TFT now have KVM capable machine on public ranch too
Quote:
```
Bare-metal instances with virtualization support for x86_64 and aarch64 in Public Ranch have landed. They will be officially announced with the upcoming 2024-03.1 release. Let us know your experience.
```

However, I wonder if we want to have that because of the rhel tests we have. WDYT @rvykydal ?

Reply:
I am not sure. Depends also on what would be the benefits.
And what would be the future use cases of running tests on tmt.
Currently we run run_travis only for master branch so it should be fine to use public, but I am not sure if it is worth (or even possible) figuring it out now.